### ISSUE TITLE
What does the 6 kb mean?

### ISSUE THREAD

**Issue by YichaoOU** [clara-parabricks/AtacWorks#235]
Hello,

In the paper, you mentioned, "The network makes predictions for each base in the genome based on coverage values from a surrounding region spanning several kilobases `6kb for the models`".

What does this 6kb really represent?

The paper mentioned "50kb non-overlapping intervals" as the input, so then the output of the model prediction is 6kb? It should also be 50kb, right?

Thanks,
Yichao

**Comment by avantikalal** [comment]
Hi Yichao, 
The model that we were describing in our paper takes 50 kb intervals as input and returns 50 kb intervals as output. 

However, the receptive field of the model was approximately 6 kb. So within that 50 kb interval, the prediction for each base depends on the bases in the ~6kb region surrounding it. This is a function of the size of the model (the number of layers, kernel width, and amount of dilation). 

In other words, the model operates on intervals of 50 kb, but the base at one end of the 50 kb interval doesn't actually influence the prediction for all the other bases in the interval.

Note that the specific values are only true for the model we were describing in that line - the interval size and model parameters can be customized if you are using AtacWorks.

Hope this helps!

**Comment by YichaoOU** [comment]
Thank you so much for the quick response!

Would you mind explain a bit more about the 6kb receptive field? Number of layers should not affect. Do you mean number of channels? Using the Kernel size of 51, and dilation of 8, I think the receptive field is 401bp?

Thanks,
Yichao

**Comment by avantikalal** [comment]
Hi Yichao, 

yes, that calculation is correct for a single layer! However, the total receptive field of the entire model is higher since there are multiple convolutional layers stacked on top of each other, and each layer adds to the receptive field.
